src/MoBi.Presentation/Presenter/EditPassiveTransportBuildingBlockPresenter.cs

```diff
@@ -48,6 +45,8 @@ public class EditPassiveTransportBuildingBlockPresenter : EditBuildingBlockPrese
          _viewItemContextMenuFactory = viewItemContextMenuFactory;
          _editTransportBuilderPresenter = editTransportBuilderPresenter;
          _transportBuilderToDTOTransportBuilderMapper = transportBuilderToDTOTransportBuilderMapper;
+         _editTransportBuilderPresenter.RemoveFormulaType<TableFormulaWithOffset>();
```

Comment:
Should be OK.
If for some reason we will need those formulas for a transport, we could still overcome this by introducing transport rate as parameter, which would have **SumFormula** or **TableFormulaWithOffset**